janusgraph-core/src/main/java/org/janusgraph/graphdb/management/JanusGraphManager.java

```diff
@@ -288,5 +272,23 @@ public Graph removeGraph(String gName) {
         if (gName == null) return null;
         return graphs.remove(gName);
     }
+
+    private void updateTraversalSource(String graphName, Graph graph){
+
```

Comment:
(nitpick) I think the typical style in JanusGraph code is not to use an empty line at the beginning of a method. So, please remove it here and also line 285.

Reply:
Removed empty lines